### ISSUE TITLE
mysql create database fails on Debian 10

### ISSUE THREAD

**Issue by pierre-b** [robertdebock/ansible-role-python_pip#6]
Hi, in order to install a Zabbix server on a Debian 10, I get stuck at the task `robertdebock.mysql : create databases]` with the following playbook

## Playbook
```yaml
---
- name: Prepare
  hosts: all
  gather_facts: yes
  become: yes

  roles:
    - role: robertdebock.bootstrap
    - role: robertdebock.buildtools
    - role: robertdebock.python_pip
      python_pip_modules: 
        - name: PyMySQL
    - role: robertdebock.container_docs
    - role: robertdebock.mysql
      mysql_databases:
        - name: zabbix
          encoding: utf8
          collation: utf8_bin
      mysql_users:
        - name: zabbix
          password: zabbix
          priv: "zabbix.*:ALL"
    - role: robertdebock.zabbix_repository
    - role: robertdebock.zabbix_server
```

## Output

TASK [robertdebock.mysql : create databases] ***********************************************************************
failed: [51.91.148.106] (item=zabbix) => {"ansible_loop_var": "item", "changed": false, "item": {"collation": "utf8_bin", "encoding": "utf8", "name": "zabbix"}, "msg": "unable to find /root/.my.cnf. Exception message: (2003, \"Can't connect to MySQL server on 'localhost' ([Errno 2] No such file or directory)\")"}

## Environment

- Control node OS: Mac OS
- Control node Ansible version: 2.9.6
- Managed node OS: Debian 10

**Comment by robertdebock** [comment]
Hi,

This role is tested against [Debian - buster/10](https://github.com/robertdebock/ansible-role-mysql/runs/724125125?check_suite_focus=true) and [others](https://travis-ci.com/github/robertdebock/ansible-role-mysql) and passes.

I'll try to reproduce locally using a Vagrant setup.

**Comment by robertdebock** [comment]
Just to update you; I'm trying to [reproduce](https://github.com/robertdebock/ansible-tester) the issue, found another one, fixing.

TL;DR: Please have a patience, I hope to reproduce & fix it soon.

**Comment by pierre-b** [comment]
Thanks for the update!
